packages/block-library/src/list/utils.js

```diff
@@ -25,7 +25,7 @@ export function createListBlockFromDOMElement( listElement ) {
 			const [ nestedList, ...nodes ] = children;
 
 			const hasNestedList =
-				nestedList.tagName === 'UL' || nestedList.tagName === 'OL';
+				nestedList?.tagName === 'UL' || nestedList?.tagName === 'OL';
```

Comment:
Strange how there's no e2e test catching this. Should we add one to test migration from an empty list? How come the integration deprecation tests are passing?

Reply:
Pushed one at https://github.com/WordPress/gutenberg/pull/43761/commits/aae46519d6e30d3bcd75b91104d0cf0ec3bf1a85